week1/todoList/index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>TodoList</title>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0" />
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet" />
+</head>
+<body>
+    <header class="logo">
+        <h2>Todo List</h2>
+    </header>
+    <nav>
+        <button>오늘만 보기</button>
+        <button>내일만 보기</button>
+        <button>함께 보기</button>
+    </nav>
+    <main>
+        <section class="left">
+            <section class="todo-header">
+                <h3 class="todo-title">오늘 할 일</h3>
+            </section>
+            <footer class="todo-add-group">
+                <input class="todo-add-group__input">
+                <button class="todo-add-group__btn-add"><span class="material-icons">&#xe145;</span></button>
+            </footer>
```

Comment:
지금 input태그랑 button태그를 통해 사용자로부터 입력을 받고, ul안에 리스트를 추가하고 있으니까 이를 감싸는 부모 태그는 
form이 적절한것같아!!


```suggestion
< form class="todo-add-group"
          method="post"
          accep-charset="utf-8"
          name="todo__list__info" >
                <input class="todo-add-group__input">
                <button class="todo-add-group__btn-add"><span class="material-icons">&#xe145;</span>. 
 </button>
</form>
```

[HTML - 폼에 대한 이해 ](https://www.nextree.co.kr/p8428/)참고해밯ㅎㅎ